### PR TITLE
[pt-br] add content/pt-br/docs/tasks/configure-pod-container/static-p…

### PR DESCRIPTION
**Pull request by DonatoHorn**

### CODE REVIEW

File: content/pt-br/docs/tasks/configure-pod-container/static-pod.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+em vez disso, o kubelet observa cada Pod estático 
+(e reinicia-os se falharem).
+
+Pods estáticos estão sempre ligados a um {{< glossary_tooltip term_id="kubelet" >}} em um nó específico.
+
+O Kubelet tenta automaticamente criar um {{< glossary_tooltip text="mirror Pod" term_id="mirror-pod" >}}
+no servidor de API do Kubernetes para cada Pod estático.
+Isso significa que os pods em execução em um nó são visíveis no servidor de API,
+mas não podem ser controlados a partir daí.
+Aos nomes de Pods será acrescido um sufixo com o nome de host do nó, com um hífem.
```
Comment (jhonmike) [inline content/pt-br/docs/tasks/configure-pod-container/static-pod.md:26]:
```suggestion
Aos nomes de Pods será sufixados com o nome de host do nó, com um hífem à esquerda. 
```
I looked at the original documentation and I think this is better and the"sufixado" exists, we can use it here

File: content/pt-br/docs/tasks/configure-pod-container/static-pod.md [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+{{< note >}}
+Se você está executando um cluster Kubernetes, usando Pods estáticos para executar um Pod em cada Nó,
+provávelmente você deveria estar usando um {{< glossary_tooltip text="DaemonSet" term_id="daemonset" >}} em substituição.
+{{< /note >}}
+
+{{< note >}}
+A `especificação` de um Pod estático não pode referir-se à outros objetos da API
+(ex., {{< glossary_tooltip text="Conta de Serviço" term_id="service-account" >}},
+{{< glossary_tooltip text="Mapa de Configuração" term_id="configmap" >}},
+{{< glossary_tooltip text="Segredo" term_id="secret" >}}, etc).
```
Comment (jhonmike) [inline content/pt-br/docs/tasks/configure-pod-container/static-pod.md:38]:
```suggestion
(ex., {{< glossary_tooltip text="ServiceAccount" term_id="service-account" >}},
{{< glossary_tooltip text="ConfigMap" term_id="configmap" >}},
{{< glossary_tooltip text="Secret" term_id="secret" >}}, etc).
```
Here I would keep kubernetes components in English

**Review by jhonmike — COMMENTED** [review]
Thank you for your translation, the text was great, just a few small suggestions I made

**Comment by rikatz** [comment]
/lgtm
/approve
Thank you!

**Pull request merged**